sites/public/src/components/listing/GetApplication.tsx

```diff
@@ -96,29 +110,18 @@ const GetApplication = (props: ApplicationsProps) => {
             {props.strings?.getAPaperApplication ?? t("listings.apply.getAPaperApplication")}
           </div>
           <Button
-            styleType={
-              !props.preview && props.onlineApplicationURL ? AppearanceStyleType.primary : undefined
```

Comment:
I think this was a bug, as far as I know there's no reason a preview version of a listing shouldn't allow you to download the paper applications.